src/main/java/ru/smirnovv/MaxBoundedTreeSet.java

```diff
@@ -5,17 +5,44 @@
 import static com.google.common.base.Preconditions.checkArgument;
 import static com.google.common.base.Preconditions.checkNotNull;
 
+/**
+ * MaxBoundedTreeSet extends the <tt>TreeSet</tt> class.
```

Comment:
A bounded TreeSet of Long numbers limited by the maximum size and intended to collect the maximum elements.